openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.activated.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:2635b9dd-0061-4059-ac61-2718ab366f75",
+  "type": "Event",
+  "actor": {
+    "id": "edx.devstack.lms:18000/u/ali",
+    "type": "Person"
+  },
+  "action": "Activated",
+  "object": {
+    "id": "course-v1:edx+cs-101+2018",
```

Comment:
`object` id should be an IRI i.e. URL. Please convert this into an example of Open edX course URL.

Reply:
fixed